extensions/scheduler/runtime/src/main/java/io/quarkus/scheduler/runtime/SimpleScheduler.java

```diff
@@ -150,6 +166,21 @@ public void resume() {
         }
     }
 
+    @Override
+    public void resume(String identity) {
+        Objects.requireNonNull(identity, "Cannot resume - identity is null");
+        if (identity.isEmpty()) {
+            LOGGER.warn("Cannot resume - identity is empty");
+            return;
+        }
+        for (ScheduledTask task : scheduledTasks) {
+            if (SchedulerUtils.lookUpPropertyValue(identity).equals(task.trigger.id)) {
```

Comment:
This will evaluate the expression on each iteration of the loop. Maybe we can evaluate it only once.

Reply:
Good point! I moved the lookup out of the loop.